### PR TITLE
[Keyboard]Add Nature Ergonomic Mechanical Keyboard

### PR DESCRIPTION
**Pull request by XBowsTech**
## Description

Add support for X-Bows Nature Ergonomic Mechanical Keyboard.
Nature, A 86 Keys Ergonomic Mechanical Keyboard with RGB backlight. It is designed according to three criteria:
*Enhancing the comfort and ergonomics of the keyboard
*Limiting the learning curve for our new layout
*Creating an attractive design

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xbows/nature/Peripherals.c [hunk 1]
```diff
@@ -0,0 +1,177 @@
+/* Copyright 2020 Shulin Huang <mumu@x-bows.com>
```
Comment (zvecr) [inline keyboards/xbows/nature/Peripherals.c:1]: This seems like a bodge to work round not going through with the requested changes within #10707?

Comment (XBowsTech) [reply keyboards/xbows/nature/Peripherals.c:1]: yes

**Comment by XBowsTech** [comment]
There seems to be an error in PR?
Do I need to modify it?

File: keyboards/xbows/nature/Peripherals.c (hunk 1 above)
Comment (zvecr) [reply keyboards/xbows/nature/Peripherals.c:1]: In which case, this should be removed then. #10707 just needed some small changes, im not sure why you closed it. This just adds fragmentation.

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by XBowsTech — COMMENTED** [review]
I may see what you mean.Let me do it again.

File: keyboards/xbows/nature/Peripherals.c (hunk 1 above)
Comment (XBowsTech) [reply keyboards/xbows/nature/Peripherals.c:1]: I modified file rgb_matrix_drivers.c  ,#10860

Comment (XBowsTech) [reply keyboards/xbows/nature/Peripherals.c:1]: If the file goes through, I will re-upload my keyboard code.

**Pull request closed without merging**